src/utils/messages.ts

```diff
@@ -2,122 +2,123 @@ import { PendingTransactionMessages, TransactionType } from '../chain-data';
 
 export const messages = {
   // Dashboard errors
-  FAILED_TO_LOAD_DASHBOARD_DATA: 'Failed to load latest dashboard data',
+  FAILED_TO_LOAD_DASHBOARD_DATA: 'Failed to load latest dashboard data. Please try again later.',
```

Comment:
> We don't really have a concept of a "dashboard" now to the end user. Might be better to make this something like:

That's a good point. I like "load staking data" but chain data works too

Reply:
I'd just use the generic one. It might be valuable for us to know which call failed, but this won't happen often and with the click history we will know how it happened.